### PR TITLE
[stm32f7] Fix RCC registers

### PR DESCRIPTION
**Pull request by maximeborges**
Following #612, fix fields  in RCC for every F7 chips

### CODE REVIEW

**Comment by adamgreig** [comment]
Could you rebase this now that #612 is merged, please?

**Comment by maximeborges** [comment]
> Could you rebase this now that #612 is merged, please?

This should be good to go.
Should I ping you when things are ready for others PRs to merge or you just don't have much time to look at them but get the notifications?

**Comment by adamgreig** [comment]
Thanks! I don't mind being pinged but do get the notifications; I've been on holiday the last week but I'm back today so will be going through these last couple of PRs in the next day or two.

File: devices/common_patches/f765_f7x7_f7x9_common.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        bitWidth: 1
+  AHB2LPENR:
+    _add:
+      JPEGLPEN:
+        description: JPEG module enabled during Sleep mode
+        bitOffset: 1
+        bitWidth: 1
+  APB1LPENR:
+    _add:
+      CAN3EN:
```
Comment (adamgreig) [inline devices/common_patches/f765_f7x7_f7x9_common.yaml:56]:
```suggestion
      CAN3LPEN:
```

**Review by adamgreig — COMMENTED** [review]
Looks good, thanks! I just noticed one typo that I'll apply the fix for now.

**Review by adamgreig — APPROVED** [review]
bors merge

**Comment by adamgreig** [comment]
ugh, bors seems to have gotten stuck?

bors cancel

**Pull request merged**